Origin: EmberBSD, AI-assisted.
mail/ruby-mime-types gains patch-lib_mime_type_columnar.rb, so bump
PKGREVISION.
Not submitted upstream.

--- a/mail/ruby-mime-types/Makefile
+++ b/mail/ruby-mime-types/Makefile
@@ -1,6 +1,7 @@
 # $NetBSD: Makefile,v 1.24 2025/05/17 07:48:56 taca Exp $
 
 DISTNAME=	mime-types-3.7.0
+PKGREVISION=	1
 CATEGORIES=	mail
 
 MAINTAINER=	pkgsrc-users@NetBSD.org
